test/unit/controllers/StartControllerSpec.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.controllers
+
+import controllers.StartController
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import unit.base.UnitSpec
+import unit.tools.Stubs._
+import views.html.start_page
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class StartControllerSpec extends UnitSpec {
+
+  val mcc = stubMessagesControllerComponents()
+  val startPage = new start_page(mainTemplate)
+
+  val controller = new StartController(validAppConfig, mcc, startPage)
+
+  "Start Controller" should {
+
+    "return 200" when {
+      
+      "display page method is invoked" in {
+
+        val result = controller.displayStartPage()(FakeRequest("", ""))
+
+        status(result) must be(OK)
```

Comment:
This is good. I like that the controller spec is now only testing what it should be testing i.e. the status return. No content as string, check what html is returned.